### ISSUE TITLE
Ask for Train Dataloader Composer

### ISSUE THREAD

**Issue by HoracceFeng** [calmevtime/DCTNet#7]
Hello @calmevtime,

Thank you for sharing this code. I am very interested in the effect of DCT toward classification and would like to do some experiments based on your code. However, I am not quite sure whether you do some special data preprocessing before training. Would you share the "composer" part you use?

BTW, I find out that in "dataloader_imagenet_dct.py", you do have "transform4" & "transform5" in the unit test part. What's the difference between them? 

Thank you.

**Comment by calmevtime** [comment]
Hi Feng,
Thanks very much for your interest. The preprocessing pipeline for validation is defined in the function valloader_upscaled_static in datasets/dataloader_imagenet_dct.py. Please ignore transform4 and transform5 because they are used for debugging only. I will delete these code in the next commit. Thanks.

**Comment by HoracceFeng** [comment]
Hello, it is me again. I use this code to train my own dataset last night, but no matter how I adjust the hyperparameters, the loss will go to NaN in less than 10 steps. 

My dataset is a traffic sign dataset, images are pretty small (around 39 to 96px). I have tried setting different batch size (64, 128 ,156), different image size (64, 128, 256, 512), and learning rate ( 1e1 to 1e4), but all of them fail to train. 

Any suggestions? 
Thank you.
